docs/source/install/installation.rst

```diff
@@ -115,28 +124,141 @@ In order to install any of these package groups, simply append them as a comma s
+
+    psql -h localhost -d aiidadb -U aiida -W
+
+and type the password you inserted before, when prompted.
+If everything worked well, you should get no error and see the prompt of the ``psql`` shell.
+
+If you uses the same names used in the example commands above, during the ``verdi setup`` phase you want to use the following parameters to use the database you just created::
+
+    Database engine: postgresql_psycopg2
+    PostgreSQL host: localhost
+    PostgreSQL port: 5432
+    AiiDA Database name: aiidadb
+    AiiDA Database user: aiida
+    AiiDA Database password: <password>
+
+.. note:: Do not forget to backup your database (instructions :ref:`here<backup_postgresql>`).
+
+.. note:: If you want to move the physical location of the data files
+  on your hard drive AFTER it has been created and filled, look at the
+  instructions :ref:`here<move_postgresql>`.
+
+
+Database setup using unix sockets
++++++++++++++++++++++++++++++++++
+
+Instead of using passwords to protect access to the database 
+(which could be used by other users on the same machine),
+PostgreSQL allows password-less logins via unix sockets. 
+
+In this scenario PostgreSQL compares the user connecting to the socket with its
+own database of users and will allow a connection if a matching user exists.
+
+Assume the role of ``postgres`` by running the following as root::
+
+    su - postgres
+
+Create a database user with the **same name** as the user you are using to run AiiDA (usually your login name)::
+
+    createuser <username>
+
+replacing ``<username>`` with your username.
+
+Next, create the database itself making sure that your user is the owner::
+
+    createdb -O <username> aiidadb
+
+To test if the database was created successfully, you can run the following command as your user in a bash terminal::
+
+    psql aiidadb
+
+
+Make sure to leave the host, port and password empty when specifiying the parameters during the ``verdi setup`` phase
+and specify your username as the *AiiDA Database user*::
+
+    Database engine: postgresql_psycopg2
+    PostgreSQL host:
+    PostgreSQL port:
+    AiiDA Database name: aiidadb
+    AiiDA Database user: <username>
+    AiiDA Database password:
+
+
+Setup instructions
+------------------
+
+After the database has been created, do
 
-or equivalently
 
 .. code-block:: bash
 
-    verdi -p <profile_name> setup
+    verdi setup <profile_name>
 
 where `<profile_name>` is a profile name of your choosing.
 The ``verdi setup`` command will guide you through the setup process through a series of prompts.
 
-The first thing that will be asked to you is the timezone, extremely important to get correct dates and times for your calculations.
+The first thing that will be asked to you is the timezone, important to get correct dates and times for your calculations.
 
 AiiDA will do its best to try and understand the local timezone (if properly configured on your machine), and will suggest a set of sensible values.
```

Comment:
Is this still true? Is it still prompting for the timezone?

Reply:
Also the section below about a single default user as the daemon runner, is outdated. Each profile has its own daemon. Also, below that, there is an example output of the prompts during the setup, which is outdated. Could you update that please?

Reply:
Ok. I will simply delete this note.
@waychal Is the aiida password currently used in the REST API?

Reply:
By the way, the output of `verdi setup` is also outdated but interactive `verdi setup` anyhow needs to be fixed (#1483). After this, the output in the docs can be updated.